BuildResidentialHPXML/measure.rb

```diff
@@ -1855,6 +1864,18 @@ def arguments(model)
     arg.setDescription('Year modules manufactured of the PV system 1.')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('pv_system_is_shared_1', true)
+    arg.setDisplayName('Photovoltaics 1: Is Shared System')
+    arg.setDescription('Whether PV system 1 is shared. If true, assumed to serve all the units in the building.')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('pv_system_building_max_power_output_1', true)
```

Comment:
I suggest removing these `pv_system_building_max_power_output` arguments and simply update the description of the `pv_system_max_power_output` arguments to clarify that the total capacity should be provided for a shared PV system.